internal/utils/helpers.go

```diff
@@ -57,3 +60,24 @@ func hasPrefix(buf []byte, prefix []byte) bool {
 func CreateWorkDir() (string, error) {
 	return CreateTmpDir(AppFS, afero.GetTempDir(AppFS, ""), "ec-work-")
 }
+
+// ImageRefTransport is used to inject the type of transport to use with the
+// remote.WithTransport function. By default, remote.DefaultTransport is
+// equivalent to http.DefaultTransport, with a reduced timeout and keep-alive
+var ImageRefTransport = remote.WithTransport(remote.DefaultTransport)
+
+// CheckImageRefAccess verifies if the provided imageRef is accessible
+func CheckImageRefAccess(imageRef string) (ok bool, err error) {
+	ref, _ := name.ParseReference(imageRef)
+	logrus.Debugf("Parsed ref: %s", ref)
+	resp, err := remote.Head(ref, ImageRefTransport)
+	if err != nil {
+		return
+	}
+	if resp == nil {
```

Comment:
I wonder if creating, and returning, a new error here would make the API a little bit simpler (e.g. callers just have to worry about checking err, not both err and ok).